crates/database/src/jit_orders.rs

```diff
@@ -0,0 +1,262 @@
+use {
+    crate::{
+        orders,
+        orders::{BuyTokenDestination, OrderKind, SellTokenSource, SigningScheme},
+        Address,
+        AppId,
+        OrderUid,
+    },
+    sqlx::{
+        types::{
+            chrono::{DateTime, Utc},
+            BigDecimal,
+        },
+        PgConnection,
+        QueryBuilder,
+    },
+};
+
+pub async fn single_full_jit_order(
```

Comment:
nit: why not
```suggestion
pub async fn fetch(
```

so it will be called via `database::jit_orders::fetch(...)` or `by_uid` or something elegant.